src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class CountryServiceImpl implements CountryService {
+    private final CountryDao countryDao = new CountryDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
Instantiating `CountryDao` in the field declaration can lead to difficulties in testing the `CountryServiceImpl` class. Consider using dependency injection to pass the `CountryDao` implementation.